app/views/casa_cases/index.html.erb

```diff
@@ -14,78 +14,9 @@
-      <button class="btn btn-secondary dropdown-toggle show" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-        Assigned to Volunteer
-      </button>
-      <div class="dropdown-menu assigned-to-volunteer-options" aria-labelledby="dropdownMenuButton">
-        <li><a class="small" data-value="option1" tabIndex="-1"><input type="checkbox" data-value="Yes" checked>Yes</a>
-        </li>
-        <li><a class="small" data-value="option1" tabIndex="-1"><input type="checkbox" data-value="No" checked>No</a>
-        </li>
-      </div>
-    </div>
-    <div class="dropdown pull-left mr-2">
-      <button class="btn btn-secondary dropdown-toggle show" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-        Assigned to more than 1 Volunteer
-      </button>
-      <div class="dropdown-menu more-than-one-volunteer-options" aria-labelledby="dropdownMenuButton">
-        <li><a class="small" data-value="option1" tabIndex="-1"><input type="checkbox" data-value="Yes" checked>Yes</a>
-        </li>
-        <li><a class="small" data-value="option1" tabIndex="-1"><input type="checkbox" data-value="No" checked>No</a>
-        </li>
-      </div>
-    </div>
-    <div class="dropdown pull-left mr-2">
-      <button class="btn btn-secondary dropdown-toggle show" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-        Assigned to Transition Aged Youth
-      </button>
-      <div class="dropdown-menu transition-youth-options" aria-labelledby="dropdownMenuButton">
-        <li>
-          <a class="small" data-value="option1" tabIndex="-1"><input type="checkbox" data-value="Yes 🦋" checked>Yes</a>
-        </li>
-        <li><a class="small" data-value="option1" tabIndex="-1"><input type="checkbox" data-value="No 🐛" checked>No</a>
-        </li>
-      </div>
-    </div>
-    <div class="dropdown pull-left mr-2">
-      <% unless current_user.volunteer? %>
-        <button class="btn btn-secondary dropdown-toggle show" type="button" id="dropdownMenuButton" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
-          Casa Case Prefix
-        </button>
-      <% end %>
-      <div class="dropdown-menu case-number-prefix-options" aria-labelledby="dropdownMenuButton">
-        <li>
-          <a class="small" data-value="option1" tabIndex="-1"><input type="checkbox" data-value="CINA" checked>CINA</a>
-        </li>
-        <li>
-          <a class="small" data-value="option1" tabIndex="-1"><input type="checkbox" data-value="TPR" checked>TPR</a>
-        </li>
-        <li>
-          <a class="small" data-value="option1" tabIndex="-1"><input type="checkbox" data-value="" checked>none</a>
-        </li>
-      </div>
-    </div>
-  </div>
-</div>
+<% if current_user.casa_admin? || current_user.supervisor?%>
+  <%= render 'filter_my_cases'%>
```

Comment:
This name is a little nonstandard, I would expect the file to be something like `app/views/casa_cases/_filters.html.erb` instead of `app/views/casa_cases/_filter_my_cases.html.erb`